packages/apps-config/src/endpoints/testingRelayRococo.ts

```diff
@@ -241,6 +241,13 @@ export function createRococo (t: TFunction): EndpointOption {
         providers: {
           'Konomi Network': 'wss://rococo.konomi.tech'
         }
+      }, {
```

Comment:
```suggestion
      },
      {
```

Have not checked, but quite possible why CI is complaining.